### ISSUE TITLE
Prefix both transition & transform for iOS Safari 8 and older

### ISSUE THREAD

**Issue by ryanheathers** [thysultan/stylis.js#22]
I'm using stylis via styled-components and I noticed a bug in iOS 8 and older. 

When using a style like this ```transition: transform 600ms ease;```, iOS Safari 8 and older requires **both** properties to be `-webkit-` prefixed.

Current stylis output:
```
-webkit-transition: transform 600ms ease;
transition: transform 600ms ease;
```

Needs to also include a double prefixed rule like this:
```
-webkit-transition: transform 600ms ease;
-webkit-transition: -webkit-transform 600ms ease;
transition: transform 600ms ease;
```

More info on this behavior: http://stackoverflow.com/questions/20669190/browser-specific-prefixes-with-a-css-transition-on-transform

Thanks!

**Comment by thysultan** [comment]
Thanks @ryanheathers will push a patch for this.

**Comment by thysultan** [comment]
This should be patched in v2.0.12

**Comment by ryanheathers** [comment]
Awesome, thanks for the fast fix!
